tools/test.sh

```diff
@@ -19,9 +19,28 @@ function prepare {
   # Webpack with the proto loader isn't used when running the tests, so the proto files need to be prepared manually
   pbjs -t json "webknossos-datastore/proto/SkeletonTracing.proto" > "$testBundlePath/SkeletonTracing.proto.json"
   pbjs -t json "webknossos-datastore/proto/VolumeTracing.proto" > "$testBundlePath/VolumeTracing.proto.json"
-  # --copy-files will copy files that are present in the source dir but are not transpiled (e.g.: json files)
-  # "$@" inside this function refers to all arguments of the prepare function, not all arguments of this bash script
-  BABEL_ENV=test babel --extensions .ts,.tsx "$jsPath" --out-dir "$testBundlePath" --copy-files "$@"
+
+  node_modules/.bin/esbuild \
+    --platform=node \
+    --format=cjs \
+    --define:process.env.BABEL_ENV=\"test\" \
```

Comment:
Not that important, but since we don't use BABEL any longer, this might be confusing. We could rename it everywhere to something more fitting. From what I can see, it is basically used as a boolean indicating whether the tests run or not, so maybe `IS_TESTING`.